environments/stg/stg.tfvars

```diff
@@ -37,6 +37,41 @@ frontends = [
     custom_domain  = "hmi-case-hq-emulator.staging.platform.hmcts.net"
     backend_domain = ["firewall-prod-int-palo-sdsstg.uksouth.cloudapp.azure.com"]
 
+    disabled_rules = {}
+  },
+  {
+    name           = "pip-publication-services"
+    custom_domain  = "pip-publication-services.staging.platform.hmcts.net"
+    backend_domain = ["firewall-prod-int-palo-sdsstg.uksouth.cloudapp.azure.com"]
+
+    disabled_rules = {}
+  },
+  {
+    name           = "pip-data-management"
```

Comment:
are all of these frontend services that need to be accessed by people directly? or are some APIs?

if they are APIs then they should just be added to the application gateway:
https://github.com/hmcts/sharedservices-azure-platform/blob/master/environments/stg/backend_lb_config.yaml

Reply:
Moved over to backend config as suggested. What should I change the DNS to point to in the other PR?